pasqal_cloud/__init__.py

```diff
@@ -230,6 +231,81 @@ def get_batch(self, id: str, fetch_results: bool = False) -> Batch:
         self.batches[batch.id] = batch
         return batch
 
+    def get_batches(
+        self,
+        filters: Optional[BatchFilters] = None,
+        pagination_params: Optional[PaginationParams] = None,
+    ) -> PaginatedResponse:
+        """
+        Retrieve a list of batches matching filters using a pagination system.
+
+        batches are sorted by their creation date in descending order.
```

Comment:
@Mildophin I just discovered the user cannot chose the ordering in the pagination params? Is that voluntary? Should that be added later?

Reply:
It was voluntary as I didn't understood it as a feature in the scope of the ticket. I think we can add it later yes.